cypress/integration/dryRunner/earyn.js

```diff
@@ -0,0 +1,46 @@
+import s from './selectors'
+
+const EARYN_DECK_ID =
+  'NU4xLDVXMSw1VzIsNU4zLDVONCw1TjUsNU42LDVONjIsNU42Niw1TjQ0LDVONDgsNVcxOQ'
+
+describe('Dry-runner — Archdruid Earyn', () => {
+  it('should be possible to play spells with Archdruid Earyn', () => {
+    const HAND = ['W1', 'N44', 'W19', 'N48']
+
+    cy.visit(`/deck/${EARYN_DECK_ID}/dry-run?mode=MANUAL`)
+
+      .drDrawHand(HAND)
+
+      .drEndTurn()
+      .drEndTurn()
+      .drEndTurn()
+      .drEndTurn()
```

Comment:
Do you think `drEndTurn` should accept a number of turns to pass (e.g. `drEndTurn(4)`)?

Reply:
It would be better to make the code clearer :)

Reply:
But I think I can try implementing it